ml-agents/mlagents/trainers/tests/test_rl_trainer.py

```diff
@@ -47,7 +47,8 @@ def create_mock_policy():
 
 @mock.patch("mlagents.trainers.rl_trainer.RLTrainer.add_policy_outputs")
 @mock.patch("mlagents.trainers.rl_trainer.RLTrainer.add_rewards_outputs")
-def test_rl_trainer(add_policy_outputs, add_rewards_outputs):
+@pytest.mark.parametrize("num_vis_obs", [0, 1], ids=["", "viz"])
```

Comment:
What does `ids=...` do here? Doesn't seem to be used.

Reply:
nm, I just RTFM'ed, looks like it just affects the test name. I've never needed to override the default behavior.

Reply:
I guess I changed it since it wouldn't be obvious what `test_rl_trainer[X]` meant